### ISSUE TITLE
doubt about Bottleneck

### ISSUE THREAD

**Issue by xuewengeophysics** [WongKinYiu/ScaledYOLOv4#131]
Dear Wong,

In my opinion, according to "CSP block" in figure4 of the paper, "conv k=3, c/2" is in front of "conv k=1, c/2", which is the opposite of what it says in the code. 
![image](https://user-images.githubusercontent.com/8676098/104404738-571e7000-5596-11eb-8f1e-64a4646fbf4e.png)

![image](https://user-images.githubusercontent.com/8676098/104404837-93ea6700-5596-11eb-892c-0a63247ee2ac.png)

Thank you very much!
----------------------------------------------
Xue Wen (问雪)
ShenZhen, China.

**Comment by WongKinYiu** [comment]
Yes, you are correct. 
We have updated the figure in github readme (https://github.com/WongKinYiu/ScaledYOLOv4#scaled-yolov4).

**Comment by xuewengeophysics** [comment]
Thank you very much!
According to the architecture figure in github readme, does the "rCSP block" correspond to "BottleneckCSP2" in the code?

![image](https://user-images.githubusercontent.com/8676098/104439034-a03ce700-55cb-11eb-9cce-7ef3e8f7903d.png)

![image](https://user-images.githubusercontent.com/8676098/104439175-c95d7780-55cb-11eb-87f0-fcbd3edd55ab.png)

![image](https://user-images.githubusercontent.com/8676098/104440442-548b3d00-55cd-11eb-82e9-d717b4847d8e.png)

**Comment by xuewengeophysics** [comment]
王老师，您好！关于ScaledYOLOv4，有几个疑问想请教您：
1、关于网络结构图中的rCSP，通过对照代码和模型配置文件yolov4-p5.yaml，我认为映射到代码中应该是BottleneckCSP2，但是代码中的BottleneckCSP2与网络结构图匹配不上？（如上面的图所示）
2、关于CSPSPP，按照我的理解，空间金字塔池化使深度卷积神经网络可以接收任意大小的输入，对区域进行分块(块数固定，例如1×1，2×2，3×3，6×6，总共50个区域)；代码中的这种写法self.m = nn.ModuleList([nn.MaxPool2d(kernel_size=x, stride=1, padding=x // 2) for x in k])，只有在图片大小相同的时候才能保证空间金字塔池化层输入固定维度的特征向量，这个和SPP的做法似乎不完全一致？
